### PR TITLE
Update the version of TAEF used in WinObjC

### PR DESCRIPTION
**Pull request by bbowman**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1707)

### CODE REVIEW

File: deps/prebuilt/TAEF/Taef.Redist.Wlk.nuspec [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
```
Comment (rajsesh-msft) [inline deps/prebuilt/TAEF/Taef.Redist.Wlk.nuspec]: I dont think we can ship this file.

**Comment by rajsesh-msft** [comment]
:clock1:

File: Frameworks/include/MockClass.h [hunk 2]
```diff
@@ -645,222 +512,17 @@ void ValidateCallback(const TFunctor& functor, const wchar_t* name) {
...
-            Test::Mock::ValidateCallback(BUILD_VARIABLE_NAME(m_, BUILD_VARIABLE_NAME(PROPERTY_NAME, get)),                    \
-                                         TAEF_WIDEN(TAEF_STRINGIZE(PROPERTY_NAME))L"::get");                                  \
-            return BUILD_VARIABLE_NAME(m_, BUILD_VARIABLE_NAME(PROPERTY_NAME, get))();                                        \
-        }                                                                                                                     \
-    }                                                                                                                         \
-    internal:                                                                                                                 \
-    void BUILD_VARIABLE_NAME(Set, BUILD_VARIABLE_NAME(PROPERTY_NAME, Getter))(                                                \
-        const decltype(BUILD_VARIABLE_NAME(m_, BUILD_VARIABLE_NAME(PROPERTY_NAME, get)))& callback) {                         \
-        BUILD_VARIABLE_NAME(m_, BUILD_VARIABLE_NAME(PROPERTY_NAME, get)) = callback;                                          \
-    }
```
Comment (jaredhms) [inline Frameworks/include/MockClass.h]: hope we never need these?

**Comment by jaredhms** [comment]
:shipit:

**Pull request closed without merging**